plugins/mobile-plugin/src/main/java/ru/sbtqa/tag/pagefactory/mobile/drivers/MobileDriverService.java

```diff
@@ -66,6 +72,10 @@ public void mountDriver() {
         mobileDriver = PROPERTIES.getAppiumPlatformName() == IOS ? new IOSDriver(url, capabilities) : new AndroidDriver(url, capabilities);
 
         deviceUdId = (String) mobileDriver.getSessionDetails().get("deviceUDID");
+
+        if (jobName != null) {
+            System.out.println(String.format("SauceOnDemandSessionID=%1$s job-name=%2$s", mobileDriver.getSessionId(), jobName));
```

Comment:
https://wiki.saucelabs.com/display/DOCS/Setting+Up+Reporting+between+Sauce+Labs+and+Jenkins
> To make sure that your test results and Sauce jobs are associated properly, you need to output the session id to stdout. For example, this is the code you would use to output the session id to the Java stdout.